scrapi/harvesters/biomed.py

```diff
@@ -15,20 +15,73 @@
 from nameparser import HumanName
 
 from scrapi import requests
-from scrapi.base import BaseHarvester
-from scrapi.linter.document import RawDocument, NormalizedDocument
+from scrapi.base import JSONHarvester
+from scrapi.base.schemas import CONSTANT
+from scrapi.linter.document import RawDocument
 
 logger = logging.getLogger(__name__)
 
 
-class BiomedHarvester(BaseHarvester):
+def process_contributors(authors):
+
+    authors = authors.strip().replace('<span class="author-names">', '').replace('</span>', '')
```

Comment:
`strip` may be a better option here if these are only at the beginning and then end.

My uncle `</span>` has been trying to get published in biomed for a while now...